src/main/java/task/Deadline.java

```diff
@@ -0,0 +1,36 @@
+package task;
+
+import java.util.List;
+
+public class Deadline extends Task {
+    private final String end;
+
+    public Deadline(String description, String end) {
+        super(description);
+        this.end = end;
+    }
+
+    public static Task parse(String taskInfo) {
+        String commandArg = "/by";
+
+        List<String> infoList = List.of(taskInfo.split(" "));
+        int commandIndex = infoList.indexOf(commandArg);
+        if (commandIndex == -1) {
```

Comment:
What does it mean by "commandIndex == -1"? Maybe can include a comment.

Reply:
Thank you for your comment. This has been resolved in 17ee986.